src/main/java/ua/lviv/iot/lab4/expression/Regular.java

```diff
@@ -0,0 +1,46 @@
+package ua.lviv.iot.lab4.expression;
+
+import java.util.Arrays;
+import java.util.Comparator;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+public class Regular {
+
+    public String switchWords() {
+        String text = "Founded on 6 March 1902 as Madrid Football Club,"
+                + " the club has traditionally worn a white home kit "
+                + "since inception. The honorific title real is Spanish"
+                + " for and was bestowed to the club by King Alfonso "
+                + "XIII in 1920 together with the royal crown in the"
+                + " emblem. The team has played its home matches in"
+                + " the 81,044-capacity Santiago Bernabeu Stadium in"
+                + " downtown Madrid since 1947. Unlike most European"
+                + " sporting entities, Real Madrid's members have "
+                + "owned and operated the club throughout its "
+                + "history.";
+        String[] arrayOfSentences = text.split("\\.");
+        String firstVowelWord;
+        StringBuilder result = new StringBuilder();
+        for (String eachSentence : arrayOfSentences) {
+            StringBuilder strb = new StringBuilder();
```

Comment:
It's not a very common practice to name your objects/variables in a shortened way (strb, mtc, etc.). It's always better to provide a full name: stringBuilder, matcher, etc.

Reply:
Thank you for your review. Naming is fixed.